Google.PowerShell/BigQuery/BqJob.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Copyright 2015-2017 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.Apis.Bigquery.v2;
+using Google.Apis.Bigquery.v2.Data;
+using Google.PowerShell.Common;
+using System;
+using System.Net;
+using System.Management.Automation;
+
+namespace Google.PowerShell.BigQuery
+{
+    /// <summary>
+    /// <para type="synopsis">
+    /// Lists all jobs that you started in the specified project or returns information about a specific job.
+    /// </para>
+    /// <para type="description">
+    /// If no Job is specified though ID parameter or object via pipeline, this cmdlet lists all jobs that 
```

Comment:
Fixed that paragraph entirely.  A lot of it was language directly from the API page, so it was stitched together somewhat awkwardly.